target_chains/ton/contracts/tests/PythTest.spec.ts

```diff
@@ -282,4 +305,198 @@ describe("PythTest", () => {
 
     // TODO: add more tests for other governance data source
   });
+
```

Comment:
i see a lot of happy paths here, can you add some tests for the failure cases? (Same for price updates)